src/pygama/lgdo/arrayofequalsizedarrays.py

```diff
@@ -46,14 +46,14 @@ def __init__(self, dims=None, nda=None, shape=None, dtype=None, fill_val=None, a
         super().__init__(nda=nda, shape=shape, dtype=dtype, fill_val=fill_val, attrs=attrs)
 
 
-    def dataype_name(self):
+    def datatype_name(self):
         """The name for this lgdo's datatype attribute"""
-        return 'array_of_equalsized_arrays'
+        return 'array_of_equal_sized_arrays'
```

Comment:
I realized this will impact reading of existing LH5 files, should I roll back @jasondet? I initially thought this was a typo...

Reply:
I reverted this change.